src/pages/explore/Explore.js

```diff
@@ -0,0 +1,44 @@
+import {NavBar} from "../../components/NavBar/NavBar";
+import { VideoCard } from "../../components/VideoCard/VideoCard";
+import { useVideo } from "../../contexts/VideoContext";
+import "./explore.css";
+
+const Explore = () => { 
+    const {videos} = useVideo();
+    return (
+        <div className="page-layout">
+            <NavBar />
+            <div className="video-listing-main flex">
+                <aside className="menu">
+                    <div className="category-menu">
+                        <div className="title">Category</div>
```

Comment:
Instead of doing that you can pass categories array that way you don't have to repeat your code as well if in the future you try to expand categories you can change in the array.

Reply:
Agree with Mayank.